### ISSUE TITLE
Ability to search or filter on named entities. Of course, a user can always search on a name like "Amsterdam" or "Thorbecke", but this could be useful to distinguish between, say, "dieren" and "Dieren" the town.

### ISSUE THREAD

**Issue by BeritJanssen** [UUDigitalHumanitieslab/I-analyzer#1610]

**Comment by JeltevanBoheemen** [comment]
Of course, a user can always search on a name like "Amsterdam" or "Thorbecke", but this could be useful to distinguish between, say, "dieren" and "Dieren" the town.
 
 N.b. 
Moved this to a comment because the issue title was overly long.

**Comment by BeritJanssen** [comment]
That's what happens when you auto-convert from a checklist.

**Comment by lukavdplas** [comment]
I think this now done, right? Though support for multi-label keyword fields is still shaky, see #1100

**Comment by BeritJanssen** [comment]
Filters are not implemented yet, so let's leave this issue open for now.
